Documentation/Signoffs/OEM Component- Auto_Light-signoff.md

```diff
@@ -0,0 +1,103 @@
+OEM Component Subsystem- Auto Light
+------
+
+## The function of the Auto Light Subsystem
+The Master microcontroller will receive the output of the Nissan Leaf AutoLight in this part of the subsystem. Furthermore, the Master microcontroller will detect if the headlight is off, on, blinking, and high and low beams.
+
+### Specification & constraints
+   - Must detect high beam,low beam, left indicator, and right indicator
+   - Must use OEM componets as the inputs
+   - The signal input must not exceed 5 V 
+   - Implement a way to protect the signal from the input voltage varying from 10 V to 15 V
+   - Current to the signal must be less than 1 percent of the current to the light
```

Comment:
I agree with the design and analysis showing this is implemented. But you need to explain this constraint (why it's here).